public/app/core/services/backend_srv.ts

```diff
@@ -4,6 +4,28 @@ import appEvents from 'app/core/app_events';
 import config from 'app/core/config';
 import { DashboardModel } from 'app/features/dashboard/state/DashboardModel';
 
+export enum HitType {
+  DashHitDB = 'dash-db',
+  DashHitHome = 'dash-home',
+  DashHitFolder = 'dash-folder',
+}
+
+export interface Hit {
```

Comment:
can we move this / merge this with the type defined here: https://github.com/grafana/grafana/blob/c1d585b156c25a76919122400d5f4c2cab9b77f4/public/app/types/search.ts

Reply:
Fixed!